.github/actions/markPullRequestsAsDeployed/markPullRequestsAsDeployed.js

```diff
@@ -34,15 +34,49 @@ function getDeployTableMessage(platformResult) {
     }
 }
 
+const androidResult = getDeployTableMessage(core.getInput('ANDROID', {required: true}));
+const desktopResult = getDeployTableMessage(core.getInput('DESKTOP', {required: true}));
+const iOSResult = getDeployTableMessage(core.getInput('IOS', {required: true}));
+const webResult = getDeployTableMessage(core.getInput('WEB', {required: true}));
+
+const workflowURL = `${process.env.GITHUB_SERVER_URL}/${process.env.GITHUB_REPOSITORY}`
+    + `/actions/runs/${process.env.GITHUB_RUN_ID}`;
+
 /**
- * Get the [added, removed] pairs for the `🔐 LockCashDeploys 🔐` label on StagingDeployCash
+ * Fetch all the StagingDeploy issues that were created after the passed fromTimestamp and
+ * including one before the fromTimestamp.
  *
+ * @param {String} fromTimestamp
+ * @returns {Promise}
+ */
+function fetchAllStagingDeployCash(fromTimestamp) {
+    return GithubUtils.octokit.paginate(GithubUtils.octokit.issues.listForRepo, {
+        owner: GithubUtils.GITHUB_OWNER,
+        repo: GithubUtils.EXPENSIFY_CASH_REPO,
+        state: 'all',
+        sort: 'created',
+        direction: 'desc',
+        labels: GithubUtils.STAGING_DEPLOY_CASH_LABEL,
+        per_page: 30,
```

Comment:
I think 30 per page is the default, right?

Reply:
yeah